### PR TITLE
Customizer: Accessibility fixes following [53024].

### PR DESCRIPTION
**Pull request by audrasjb**
Trac ticket: https://core.trac.wordpress.org/ticket/54939

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/script-loader.php [hunk 1]
```diff
@@ -1223,15 +1222,13 @@ function wp_default_scripts( $scripts ) {
 			'invalidDate'             => __( 'Invalid date.' ),
 			'invalidValue'            => __( 'Invalid value.' ),
 			'blockThemeNotification'  => sprintf(
-				/* translators: 1. %s: URL to the localized version of https://wordpress.org/support/article/site-editor/, 2: HTML Button. */
-				__( 'Hurray! Your theme supports Full Site Editing with blocks. <a href="%1$s" target="_blank">Tell me more</a>. %2$s' ),
+				/* translators: 1: HelpHub link to Site Editor documentation, 2: HTML Button. */
+				__( 'Hurray! Your theme supports Full Site Editing with blocks. <a href="%1$s">Tell me more</a>. %2$s' ),
 				__( 'https://wordpress.org/support/article/site-editor/' ),
 				sprintf(
-					/* translators: 1: URL to the Site Editor admin screen, 2: Button label, 3: Button text. */
```
Comment (pbiron) [inline src/wp-includes/script-loader.php:1230]: Shouldn't this translators comment be left in (with `3: Button text` removed)?

Comment (pbiron) [reply src/wp-includes/script-loader.php:1230]: D'oh!  of course not, my bad.

Comment (audrasjb) [reply src/wp-includes/script-loader.php:1230]: 👍  :)

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/53025

**Pull request closed without merging**